### PR TITLE
Add Listrak

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: img/l/listrak.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><path fill="#FFF" d="M0 0h90v90H0z"/><path fill="#010101" d="M88.47 33.48c-6.34-4.43-17.08-6.54-27.25-7.5 13.62-9.24 19.38-8.09 19.38-8.09C73.7 9.62 68.32 7.12 68.32 7.12c-7.86.58-18.03 7.5-25.7 13.85 3.26-16.74 8.63-20 8.63-20a47.44 47.44 0 00-16.5.57c-3.07 3.27-5.18 8.66-6.52 14.43l-8.06-7.89A43.48 43.48 0 007.7 20.01l8.44 8.27C4.05 31.17 1.18 35.02 1.18 35.02A38.33 38.33 0 00.6 51.18c3.07-4.43 11.9-7.12 19.95-8.47-14.39 17.32-14.2 25.4-14.2 25.4A46.8 46.8 0 0017.3 80.23c0-5.38 3.65-12.5 8.25-19.04 1.92 19.24 7.67 26.74 7.67 26.74 5.38 1.35 10.94 1.92 16.5 1.35-6.71-7.5-8.63-25.79-9.2-36.37l29.35 28.86a46.06 46.06 0 0012.27-12.12L53.36 40.8c29.16 1.15 36.26 9.04 36.26 9.04.76-5.38.38-10.96-1.15-16.35z"/></svg>
```
Comment (ApCoder123) [inline img/l/listrak.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><path fill="#010101" d="M88.47 33.48c-6.34-4.43-17.08-6.54-27.25-7.5 13.62-9.24 19.38-8.09 19.38-8.09C73.7 9.62 68.32 7.12 68.32 7.12c-7.86.58-18.03 7.5-25.7 13.85 3.26-16.74 8.63-20 8.63-20a47.44 47.44 0 00-16.5.57c-3.07 3.27-5.18 8.66-6.52 14.43l-8.06-7.89A43.48 43.48 0 007.7 20.01l8.44 8.27C4.05 31.17 1.18 35.02 1.18 35.02A38.33 38.33 0 00.6 51.18c3.07-4.43 11.9-7.12 19.95-8.47-14.39 17.32-14.2 25.4-14.2 25.4A46.8 46.8 0 0017.3 80.23c0-5.38 3.65-12.5 8.25-19.04 1.92 19.24 7.67 26.74 7.67 26.74 5.38 1.35 10.94 1.92 16.5 1.35-6.71-7.5-8.63-25.79-9.2-36.37l29.35 28.86a46.06 46.06 0 0012.27-12.12L53.36 40.8c29.16 1.15 36.26 9.04 36.26 9.04.76-5.38.38-10.96-1.15-16.35z"/></svg>
```
Please remove the white background on the icon.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by 40504** [comment]
I think is better to list black logos with a background related to the service, because on devices with dark mode on that logos are invisible.

**Comment by ApCoder123** [comment]
On the site, even when dark mode is enabled, the table's background is still green. Please could you provide more info as to when the logos may appear invisible on dark mode?

**Review by ApCoder123 — APPROVED** [review]

**Comment by 40504** [comment]
I think you are aware that information provided by the `2fa.directory` is used in many products like passwords managers and authenticator apps. In our Authenticator app we have a list of services (from the 2fa.directory) that support totp option, and grab domain names and images for them. That why dark logos are hard or even can't be seen on Apple devices with dark mode on and white logos on light mode. I know, is our problem but if you already take care of the additional domains option, that is very useful on password managers, why not apply some background color for images, where is possible and/or is needed.

By the way, thank you to everyone that this repository exists.

**Comment by ApCoder123** [comment]
I see what you mean - maybe instead of adding a white background to this logo, we can change the colour to `#0b1c40`. They use that colour on their social media profiles - this should be a bit more visible on darker backgrounds 
image
but will still keep it looking clean on 2fa.directory 
image
What do you think? If this is still too hard to see, I am not opposed to re-adding the white background.

Thank you for all the contributions you have made to the repositiory.

**Comment by 40504** [comment]
That's much better than flat dark. @ApCoder123  👍

**Comment by Carlgo11** [comment]
> I think you are aware that information provided by the `2fa.directory` is used in many products like passwords managers and authenticator apps. In our Authenticator app we have a list of services (from the 2fa.directory) that support totp option, and grab domain names and images for them. That why dark logos are hard or even can't be seen on Apple devices with dark mode on and white logos on light mode. I know, is our problem but if you already take care of the additional domains option, that is very useful on password managers, why not apply some background color for images, where is possible and/or is needed.
> 
> By the way, thank you to everyone that this repository exists.

@40504 you're aware there's an attribution requirement when using the data, right? Because there's been a lot of companies that haven't realized that.

**Review by Carlgo11 — APPROVED** [review]

**Comment by 40504** [comment]
@Carlgo11 fully agrees with you. Even 1Password didn't do it :).
I use 1Password and can't find any mention of this repository 😞

**Pull request merged**